packages/firestore/e2e/Aggregate/AggregateQuery.e2e.js

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2022-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const COLLECTION = 'firestore';
+const { wipe } = require('../helpers');
+describe('getAggregateFromServer()', function () {
+  before(function () {
+    return wipe();
+  });
```

Comment:
wipe is async - this one (and the `count.e2e.js` e2e file next to this new `AggregateQuery.e2e.js` one should use it async like in other areas https://github.com/invertase/react-native-firebase/blob/bb76c4d802890855cb7b1f151f422bea28101715/packages/firestore/e2e/Bundle/loadBundle.e2e.js#L25-L27

```suggestion
  before(async function () {
    return async wipe();
  });
```